interfaces/inc/c_pal/threadpool.h

```diff
@@ -21,10 +21,13 @@ extern "C" {
 #endif
 
 typedef struct THREADPOOL_TAG THREADPOOL;
+typedef struct TIMER_INSTANCE_TAG TIMER_INSTANCE;
```

Comment:
>TIMER_INSTANCE [](http://example.com/codeflow?start=34&length=14)

poetry: I would normally say that _INSTANCE doe snot add any value. THANDLE(TIMER) should be good enough. BUt this being also a very rippling change, I think it can be done (if considered) as a separate task.

Reply:
done renamed to TIMER